software/edge/src/aeris_device_manager/aeris_device_manager/device_manager_node.py

```
@@ -32,48 +37,86 @@ class DeviceManagerNode(Node):
 
         queue_depth = int(self.declare_parameter("queue_depth", 10).value)
-        poll_period_sec = float(
-            self.declare_parameter("poll_period_sec", 0.5).value
-        )
+        poll_period_sec = float(self.declare_parameter("poll_period_sec", 0.5).value)
         enumeration_budget_sec = float(
             self.declare_parameter("enumeration_budget_sec", 15.0).value
         )
+        registry_path_param = self.declare_parameter(
+            "inventory_registry_path",
+            "/tmp/aeris_device_manager_pod_inventory.json",
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/aeris_device_manager_pod_inventory.json"

(S108)